tracer/build/_build/Build.Steps.cs

```diff
@@ -602,6 +603,18 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
                 var workingDirectory = ArtifactsDirectory / $"linux-{LinuxArchitectureIdentifier}";
                 EnsureCleanDirectory(workingDirectory);
 
+                if (!IsArm64)
+                {
+                    var tracerNativeFile = MonitoringHomeDirectory / "Datadog.Trace.ClrProfiler.Native.so";
+                    var newTracerNativeFile = MonitoringHomeDirectory / "tracer" / "Datadog.Tracer.Native.so";
```

Comment:
The profiler native binary is called: `Datadog.AutoInstrumentation.Profiler.Native.{architecture}.dll`
The tracer native binary is called: `Datadog.Trace.ClrProfiler.Native.so`

I feel like we should unify those names before we ship this, one way or another. I'm not passionate either way, but I think we should pick one

My _only_ concern about the `*AutoInstrumentation*` section, is the name starts to get long, and we don't want to be bitten by `MAX_PATH` 😅

Reply:
TBH I do not like the `*AutoInstrumentation*` too :P (and one of the reasons is the one you pointed out.... I got bitten... already)
I will go for:
Linux
`Datadog.Profiler.Native.so` and `Datadog.Tracer.Native.so`
Windows
`Datadog.Profiler.Native.{architecture}.dll` and `Datadog.Tracer.Native.{architecture}.dll`

Reply:
LGTM in general, though I wonder if we should have the `{architecture}` in the .so too 🤔 Might be necessary when we look at the dotnet-tool layout (maybe we _should_ do that in this PR too?)

Reply:
@zacharycmontoya any thoughts ?

Reply:
Sorry, I'm just getting around to this 😬 I like the `Datadog.<Product>` convention. I don't feel strongly about having the architecture in the name because for all Linux distro's we're only building 64-bit DLL's and we can also distinguish Windows 32-bit/64-bit by folder, like we do today

Reply:
>  I don't feel strongly about having the architecture in the name because for all Linux distro's we're only building 64-bit DLL's 

Well, we _do_ build both `arm64` and `x64` dlls...

Reply:
🤦🏼 I'm not thinking straight today...

Reply:
Ok after working on it for a while, renaming the libraries (more than what I currently did) will be painful and will make this PR too big.
We need to package the .NET Profiler with the Tracer for a public beta release by the end of the month.
I would merge it is and create PRs to address the different issues:
- Profiler mapping for P/Invoke must be rewritten (like what's already done for the Tracer, AppSec)
- Rename libraries with the convention mentioned above `Datadog.<Product>.<Native|Managed>.<arch>.<extension>` the `<arch>` will be discussed in the PR